frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/components/ComponentPortlet.java

```diff
@@ -587,10 +589,26 @@ private void loadSpdxLicenseInfo(ResourceRequest request, ResourceResponse respo
                     }
                 });
                 jsonGenerator.writeEndArray();
-            } else if (licenseWithText != null) {
+            } else if (mainLicenseNames.size() > 0) {
```

Comment:
`CommonUtils.isNotEmpty(mainLicenseNames)` would look good here.

Reply:
added the requested change.